server/queries/db.js

```diff
@@ -76,6 +78,57 @@ module.exports.addLabel = (cardId, word) => {
     });
 }
 
+module.exports.addCard = (etag, url) => {
+    return new Promise((resolve, reject) => {
```

Comment:
this is an unnecessary promise.
`CardImages.findOrCreate()` already returns a thenable, so you should chain from that

```js
CardImages.findOrCreate(/* whatever */)
.then(([card, created]) => {
  if (created) return card;
  throw {
    code: 409,
    message: "Card already exists in the database"
  };
})
```

Reply:
but then I want addCard to return a thenable too